src/utils/gaa.js

```diff
@@ -157,7 +157,7 @@ const REGWALL_HTML = `
   .gaa-metering-regwall--iframe {
     border: none !important;
     display: block !important;
-    height: 36px !important;
+    height: 44px !important;
```

Comment:
Out of curiosity, why this change in height please?

Reply:
This is because the default height for the button is 44px.
I was previously having an issue in which the bottom edge of the button was being cut off. Changing this height resolved the issue.